src/main/java/gov/cms/mat/cql_elm_translation/controllers/MatMeasureController.java

```diff
@@ -0,0 +1,84 @@
+package gov.cms.mat.cql_elm_translation.controllers;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.cqframework.cql.cql2elm.LibraryBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import gov.cms.mat.cql.dto.CqlConversionPayload;
+import gov.cms.mat.cql_elm_translation.controllers.CqlConversionController.TranslatorOptionsRemover;
+import gov.cms.mat.cql_elm_translation.data.RequestData;
+import gov.cms.mat.cql_elm_translation.service.CqlConversionService;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+@RequestMapping(path = "/mat/translator")
+@Tag(name = "Conversion-Controller", description = "API for converting MAT measure CQL to ELM.")
```

Comment:
"Conversion" is already heavily used by the community to indicate a measure model change: QDM to FHIR, FHIR to QICore, etc.

Could we go with "Transfer" or "Measure-Import" Controller, instead?

Reply:
@jkotanchik-SB Ok, sure.